src/geode/io/mesh/vtp_input.cpp

```diff
@@ -390,11 +380,12 @@ namespace
             std::istringstream iss( data.data() );
```

Comment:
Maybe you don't need an iss with split functions https://github.com/abseil/abseil-cpp/blob/master/absl/strings/str_split.h#L493

Reply:
https://github.com/abseil/abseil-cpp/blob/master/absl/strings/str_split.h#L475